### DISCUSSION TITLE
what is the max number of items you can create with createMany?

### DISCUSSION THREAD

**Discussion by yanndebelgique** [prisma/prisma#13916] [Q&A]
want to know what limit is of items I can create with this

**Comment by austincrim** [comment]
Hey @yanndebelgique 👋,

There is no limit on the Prisma side. We will chunk up the updates and send them over based on the parameter limits of the given database.

You might hit limits on the computation or database side, though, depending on how high you are going.

Let us know if you do some experimenting!

**Reply by dev-dionmar** [reply]
is it possible to pass a value to the nested table while  using createMany ?

**Reply by janpio** [reply]
No, see the request about this here: https://github.com/prisma/prisma/issues/5455

(Please keep comments on existing discussions on topic. Your question here is not about what the original discussion about, so often we just do not see such follow up questions. Please just open a new discussion. Thanks.)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
